generators/environment/templates/bin/start.sh

```diff
@@ -9,154 +9,6 @@
-# docker manifest which will be used by default. With it set, no -f flag is
-# needed.
-#
-# We use it in this script to declutter the majority of docker-compose
-# commands--which use the build container.
-#
-# Instead of setting this environment variable, you could invoke commands with:
-#
-# docker-compose -f build.yml <args>
-export COMPOSE_FILE=build$COMPOSE_EXT.yml
-
-##
-# Error handling.
-##
-
-# Stop all containers.
-# This will allow follow-up runs via Jenkins to manage the workspace.
-# Container rm is left to Jenkins or the developer.
-teardown() {
-  docker-compose -f docker-compose.yml stop || true
-}
-
-# Handler for errors and interruptions.
-cancel() {
-  echoError "$NAME: Error: Line $1: $2\n"
-  teardown
-  exit 33
-}
-
-# Final actions to take whenever the script ends.
-complete() {
-  ret=$1
-  [ "$ret" -eq 0 ] || echoFail >&2 "$NAME: aborted ($ret)\n"
-}
-
-# Cancel docker start on errors.
-trap 'cancel $LINENO $BASH_COMMAND' ERR SIGINT SIGTERM SIGQUIT
-trap 'complete $?' EXIT
-
-##
-# Bring up the site.
-##
-echoInfo "Preparing site for environment '$DOCKER_ENV' using start.sh (v${START_VERSION})\n"
-export DOCKER_ENV
-
-# Spin up cache and db services to support build container.
-# Web container might take file locks on existing code, blocking the build process.
-cmd "docker-compose -f docker-compose.yml up -d <% if(cache.external) { %>cache <% } %>db"
-
-# Install dependencies and run main application build.
-# Run grunt with --force to ignore errors. This won't help if the errors bail the build.
-echoInfo "When grunt commands fail they suggest running with '--force'.\n"
-echoInfo "This just suppresses errors and is unlikely to fix issues causing your build process to fail.\n"
-cmd "docker-compose run --rm -e NPM_CONFIG_LOGLEVEL=error cli \"npm install && grunt --timer --quiet${NO_VALIDATE}\""
-
-# Now safe to activate web container to support end-to-end testing.
-cmd "docker-compose -f docker-compose.yml up -d <% if(proxy.exists) { %>proxy <% } %>www"
-
-# Correct any issues in the web container.
-cmd "docker exec <%= machineName %>_${DOCKER_ENV}_www \"/var/www/bin/fix-perms.sh\""
-
-# Install the site.
-if [ "$UPDATE" == 0 ]; then
-  if [ ! -z "$FORCE" ]; then
-    echoInfo "The start.sh --force flag was used, site install will proceed despite any errors.\n"
-  fi
-  cmd "docker-compose run --rm grunt \"install --no-db-load${FORCE}\""
-else
-  echoInfo "'grunt update' is defined in Gruntconfig.json\n"
-  cmd "docker-compose run --rm grunt update"
-fi
-
-# Wipe cache after permissions fix.
-cmd "docker-compose run --rm grunt cache-clear"
-
-echo
-echoSuccess "Application Setup Complete: "
-URL=$(docker-compose -f build$COMPOSE_EXT.yml run --rm drush sa @<%= projectName %> --format=list --fields=uri)
-echo "$URL"
-echo
+echoWarn "Outrigger Drupal v4 dropped support for the start.sh script utility."
+echoWarn "For local development, replace your daily operations with rig project commands."
```

Comment:
Might be nice to indicate which one is the corollary of start.sh functionality (setup it seems) and a pointer that many more discrete commands are available.

Reply:
Something like this:

```
echoWarn "Outrigger Drupal v4 dropped support for the start.sh script utility."
echoWarn "You may instead use 'rig project setup' for your start.sh needs."
echoWarn "Run 'rig project' to review the more specific commands available, as well as direct use of the underlying tools such as composer."
```

Reply:
I like it.